README.md

```diff
@@ -603,7 +603,7 @@ If you're using [Spring](https://github.com/rails/spring) to speed up test suite
 
     ```ruby
     # Disable Rails's static asset server (Apache or nginx will already do this)
-    config.serve_static_files = false
+    config.public_file_server.enabled = false
```

Comment:
Make sense. I've updated the readme for rails 5 and earlier versions. Also note that I've removed the config in `development.rb`, because I think sometimes we need the assets in the public folder when using development server. Or there are some special reasons that we should add it to `development.rb`? Let me know if there is something wrong.